sdk/eventhub/Azure.Messaging.EventHubs/tests/Diagnostics/DiagnosticsLiveTests.cs

```diff
@@ -0,0 +1,555 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Core;
+using Azure.Messaging.EventHubs.Errors;
+using Azure.Messaging.EventHubs.Metadata;
+using Azure.Messaging.EventHubs.Tests.Infrastructure;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Tests
+{
+    /// <summary>
+    ///   The suite of live tests for the <see cref="TrackOne.EventHubsDiagnosticSource" />
+    ///   class.
+    /// </summary>
+    ///
+    /// <remarks>
+    ///   These tests have a depenency on live Azure services and may
+    ///   incur costs for the associated Azure subscription.
+    /// </remarks>
+    ///
+    [TestFixture]
+    [NonParallelizable]
+    [Category(TestCategory.Live)]
+    [Category(TestCategory.DisallowVisualStudioLiveUnitTesting)]
+    class DiagnosticsLiveTests
+    {
+        /// <summary>The maximum number of times that the receive loop should iterate to collect the expected number of messages.</summary>
+        private const int ReceiveRetryLimit = 10;
+
+        private static ConcurrentQueue<(string eventName, object payload, Activity activity)> CreateEventQueue() =>
```

Comment:
nit:  If you wouldn't mind, I'd appreciate if we could XML doc all of the members for consistency with our track two approach.